daemons/controld/controld_remote_ra.c

```diff
@@ -629,7 +629,7 @@ remote_lrm_op_callback(lrmd_event_data_t * op)
                           cmd->remaining_timeout);
                 pcmk__set_result(&(cmd->result), PCMK_OCF_UNKNOWN_ERROR,
                                  PCMK_EXEC_TIMEOUT,
-                                 pcmk_strerror(op->connection_rc));
+                                 "%s", pcmk_strerror(op->connection_rc));
```

Comment:
or simply use crm_strdup_printf wherever needed together as argument of pcmk__set_result ...
But pcmk__set_formatted_result would be shorter of course.

Reply:
edited commits to add new functions instead of modify existing ones

crm_strdup_printf() would have been just as ugly b/c it would still require the if-NULL-else blocks, plus a free()